### ISSUE TITLE
Add OSGI METADATA to support deployment in OSGI container

### ISSUE THREAD

**Issue by cmoulliard** [logstash/log4j-jsonevent-layout#31]
The log4j-jsonevent-layout does not include OSGI META-DATA required to be deployed in an OSGI container (Apache Felix, Karaf, ...). 

Suggestion : Add a MANIFEST as described here (example combining log4j - nagios : http://java.dzone.com/articles/integrate-log4j-nagios)

**Comment by cmoulliard** [comment]
Profile to be added to the pom

```
    <profile>
      <id>bundle</id>
      <build>
        <plugins>
          <plugin>
              <groupId>org.apache.felix</groupId>
              <artifactId>maven-bundle-plugin</artifactId>
              <version>2.3.7</version>
              <extensions>true</extensions>
              <configuration>
                  <instructions>
                    <Bundle-Name>${project.groupId}.${project.artifactId}</Bundle-Name>
                    <Bundle-SymbolicName>${project.groupId}.${project.artifactId}</Bundle-SymbolicName>
                    <Import-Package>!*</Import-Package>
                    <Fragment-Host>org.ops4j.pax.logging.pax-logging-service;bundle-version="[1.6,1.7)"</Fragment-Host>
                    <Embed-Dependency>*;scope=compile|runtime;inline=true</Embed-Dependency>
                    <Implementation-Version>${project.version}</Implementation-Version>
                  </instructions>
              </configuration>
          </plugin>
        </plugins>
      </build> 
    </profile> 
```

Remark : version number should be changed according to the build release

**Comment by lusis** [comment]
I'll give this a read. MY OSGI foo is a bit weak but this doesn't pull in any additional libraries does it? It's just metadata added to the final artifact?

**Comment by lusis** [comment]
added in 476ffd5923059275651eb4c4188a15cf7cf7c3c5

I'm hoping to cut a 1.6 later today with some additional changes.

**Comment by vkadam** [comment]
I see the profile named bundle added to pom.xml, but its not activated thats why MANIFEST.MF does not contain the OSGI properties.

**Comment by vkadam** [comment]
Fixed in #55
